### ISSUE TITLE
Failed to scale a DC with oc

### ISSUE THREAD

**Issue by kargakis** [openshift/origin#17729]
```
$ oc scale dc/hook --replicas=0
error: Scaling the resource failed with: the API version in the data (extensions/v1beta1) does not match the expected API version (apps.openshift.io/v1); Current resource version Unknown
$ oc version
oc v3.9.0-alpha.0+713ad46-76
kubernetes v1.8.1+0d5291c
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://api.ci.openshift.org:443
openshift v3.7.0-rc.0+bd64aab-99
kubernetes v1.7.6+a08f5eeb62
```
/kind bug
/sig master

@openshift/sig-master

**Comment by liggitt** [comment]
fixed in 3.7.x in https://github.com/openshift/origin/pull/17517

**Comment by kargakis** [comment]
@liggitt this is happening with an `oc` built from master (I just tested with `v3.9.0-alpha.0+4d15c88-84`). Assuming 3.8 is not a release that will be shipped, shouldn't 3.9 clients work against 3.7 masters?

**Comment by kargakis** [comment]
Ok, I think we just need to update our cluster in order for this to work.

**Comment by liggitt** [comment]
3.6 and 3.7 shipped with the scale subresource under apps.openshift.io/v1 broken. It is fixed in the 3.6.x, 3.7.x, 3.8.x and master branches.
